boom! testing rain-0.2.2 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! 99b302ef0e8eb7016f7a16708a204422e915eb912c02c80ec9ad6d9b7556ed5a
boom! running `docker start -a 99b302ef0e8eb7016f7a16708a204422e915eb912c02c80ec9ad6d9b7556ed5a`
kablam!    Compiling rain v0.2.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 1.62 secs
boom! running `docker rm -f 99b302ef0e8eb7016f7a16708a204422e915eb912c02c80ec9ad6d9b7556ed5a`
blam! 99b302ef0e8eb7016f7a16708a204422e915eb912c02c80ec9ad6d9b7556ed5a
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 8cd2cb3b476f95530450524eef28407d5c1115cd226ba82b39ffd7f7bbc59c09
boom! running `docker start -a 8cd2cb3b476f95530450524eef28407d5c1115cd226ba82b39ffd7f7bbc59c09`
kablam!    Compiling rain v0.2.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 5.71 secs
boom! running `docker rm -f 8cd2cb3b476f95530450524eef28407d5c1115cd226ba82b39ffd7f7bbc59c09`
blam! 8cd2cb3b476f95530450524eef28407d5c1115cd226ba82b39ffd7f7bbc59c09
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! 2cf5ddcba6a4c1f0e01e13d4e5226008a56eda2ac9962a51c16951710df30c41
boom! running `docker start -a 2cf5ddcba6a4c1f0e01e13d4e5226008a56eda2ac9962a51c16951710df30c41`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.10 secs
kablam!      Running /target/debug/deps/error-c916352a329f1f90
blam! 
blam! running 1 test
blam! test success_convert_from_io_error ... ok
blam! 
blam! test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!      Running /target/debug/deps/lib-f303e384dcf104e7
blam! 
blam! running 8 tests
blam! test add_remove_success_4 ... FAILED
blam! test add_remove_success_3 ... FAILED
blam! test add_remove_success_2 ... FAILED
blam! test add_remove_success_1 ... FAILED
blam! test add_remove_success_signed_integer ... FAILED
blam! test print_if_new_data_success ... FAILED
blam! test random_add_remove_success ... FAILED
blam! test print_long_prefix_success ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- add_remove_success_4 stdout ----
blam! 	thread 'add_remove_success_4' panicked at 'assertion failed: graph.print().is_ok()', tests/lib.rs:71
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- add_remove_success_3 stdout ----
blam! 	thread 'add_remove_success_3' panicked at 'assertion failed: graph.print().is_ok()', tests/lib.rs:57
blam! 
blam! ---- add_remove_success_2 stdout ----
blam! 	thread 'add_remove_success_2' panicked at 'assertion failed: graph.print().is_ok()', tests/lib.rs:47
blam! 
blam! ---- add_remove_success_1 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'add_remove_success_1' panicked at 'assertion failed: graph.print().is_ok()', tests/lib.rs:16
blam! 
blam! ---- add_remove_success_signed_integer stdout ----
blam! 	thread 'add_remove_success_signed_integer' panicked at 'assertion failed: graph.print().is_ok()', tests/lib.rs:145
blam! 
blam! ---- print_if_new_data_success stdout ----
blam! 	thread 'print_if_new_data_success' panicked at 'called `Result::unwrap()` on an `Err` value: Code: Other, Description: Unable to get the terminal size.', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- random_add_remove_success stdout ----
blam! 	thread 'random_add_remove_success' panicked at 'assertion failed: graph.print_if_new_data().is_ok()', tests/lib.rs:133
blam! 
blam! ---- print_long_prefix_success stdout ----
blam! 	thread 'print_long_prefix_success' panicked at 'assertion failed: graph.print().is_ok()', tests/lib.rs:90
blam! 
blam! 
blam! failures:
blam!     add_remove_success_1
blam!     add_remove_success_2
blam!     add_remove_success_3
blam!     add_remove_success_4
blam!     add_remove_success_signed_integer
blam!     print_if_new_data_success
blam!     print_long_prefix_success
blam!     random_add_remove_success
blam! 
blam! test result: FAILED. 0 passed; 8 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 2cf5ddcba6a4c1f0e01e13d4e5226008a56eda2ac9962a51c16951710df30c41`
blam! 2cf5ddcba6a4c1f0e01e13d4e5226008a56eda2ac9962a51c16951710df30c41
